src/components/TransactionModal.tsx

```diff
@@ -42,8 +48,11 @@ const PendingTransactionModalBody = () => {
         </h2>
         <p>
           <Trans>
-            Your transaction has been submitted and is awaiting confirmation.
-          </Trans>
+            Your transaction has been submitted and is awaiting confirmation{' '}
+          </Trans>{' '}
+          {pendingTx ? (
+            <EtherscanLink value={pendingTx.tx?.hash} type="tx" />
```

Comment:
**suggestion** wdyt about a text link "Go to Etherscan" or "View in Etherscan" instead?

Reply:
yup, i've chosen "View in Etherscan"

updated in the latest commit ser 🍻

Reply:
![image](https://user-images.githubusercontent.com/18723426/198209127-c937f3c6-0be0-4923-ba65-d22dc0aac6cd.png)